ux.symfony.com/templates/base.html.twig

```diff
@@ -9,66 +9,45 @@
         <link rel="manifest" href="/site.webmanifest" />
         <link rel="mask-icon" href="https://symfony.com/favicons/safari-pinned-tab.svg" color="#5bbad5">
         <meta name="viewport" content="width=device-width, initial-scale=1">
+        <meta name="color-scheme" content="light dark">
+        <link rel="preconnect" href="https://fonts.googleapis.com" />
```

Comment:
Really, we need to remove google fonts - it's not typically trusted from a privacy pov. That can be here or a different PR (obviously, you didn't add google fonts in this PR). An easy option is to use fontsource instead via jsdelivr - e.g. https://cdn.jsdelivr.net/npm/@fontsource-variable/inter@5.0.8 (note: jsdelivr has the new `@fontsource-variable` packages, but due to a bug in npm's search API, they won't show up if you search for these packages through the jsdelivr web interface).

Reply:
My personal opinion on that matter would be: we should not use any webfont * 

* okay.. let's agree to use "one" :) 

SF Mono is a superb monospace font (don't know how it is on android but I trust typo experts at Google, Microsoft, Canonical, ... to embed high quality system fonts),

Playfair could be replaced by well-crafted SVG texts, and Inter .. **if** we want to stay with it.. is available as variable webfont in better quality/compression outside Google Fonts.

Reply:
I suggest to work on that question on a following PR ... you can trust me to not forget :)

Reply:
Good for me!